### DISCUSSION TITLE
raspicat_documentation/chat_corner/request/

### DISCUSSION THREAD

**Discussion by giscus** [CIT-Autonomous-Robot-Lab/raspicat_documentation#22] [Announcements]
# raspicat_documentation/chat_corner/request/

https://uhobeike.github.io/raspicat_documentation/chat_corner/request/

**Comment by TonoLeo** [comment]
@uhobeike リモートでマッピングをしたときはmap_severの保存場所をraspi側で行ってくだいさい

**Comment by TonoLeo** [comment]
raspi4のみでマッピングを行ったあとにマップを保存する場所がraspiではなくノートpcになってます！！

**Reply by uhobeike** [reply]
あざす🙏

**Comment by TonoLeo** [comment]
raspi4のみでのマッピング時のmap_servserになってます。map_serverです！！

**Reply by uhobeike** [reply]
あざす🙏

**Comment by TonoLeo** [comment]
マッピングのページにコントローラを起動するコマンドの
roslaunch raspicat_gamepad_controller logicool.launch
を加えてください

**Comment by TonoLeo** [comment]
waypointを設置する時にroslaunchを起動する前にroscoreするのを入れたほうがいいと思います
